Sources/SwiftProtobuf/Google_Protobuf_Struct.swift

```diff
@@ -248,7 +248,10 @@ public struct Google_Protobuf_Value: Message, Proto3Message, _MessageImplementat
     mutating public func _protoc_generated_decodeField<T: Decoder>(decoder: inout T, fieldNumber: Int) throws {
         switch fieldNumber {
         case 1, 2, 3, 4, 5, 6:
-            try kind.decodeField(decoder: &decoder, fieldNumber: fieldNumber)
+            if kind != nil {
```

Comment:
If I remember correctly, the C++ text decoder optionally rejects conflicting one of fields.  If we need to support such an option someday, `handleConflictingOneOf()` can conditionally throw.